lib/logstash/inputs/gelf.rb

```diff
@@ -206,8 +206,17 @@ def udp_listener(output_queue)
       event = self.class.new_event(data, client[3])
       next if event.nil?
 
-      remap_gelf(event) if @remap
-      strip_leading_underscore(event) if @strip_leading_underscore
+      begin
+        remap_gelf(event) if @remap
+        strip_leading_underscore(event) if @strip_leading_underscore
+      rescue => e
+        if !stop?
+          @logger.error("Caught exception while striping leading underscores", :exception => e)
```

Comment:
I think that here we should be more consistent with the TCP counter part, so I would output  `warn` log in form:
```
@logger.warn("Gelf (udp): striping leading underscores failed.", :exception => ex, :backtrace => ex.backtrace
```